tpch/queries/q15.sql

```diff
@@ -0,0 +1,33 @@
+-- SQLBench-H query 15 derived from TPC-H query 15 under the terms of the TPC Fair Use Policy.
+-- TPC-H queries are Copyright 1993-2022 Transaction Processing Performance Council.
+create view revenue0 (supplier_no, total_revenue) as
```

Comment:
I got:

```
pyspark.errors.exceptions.captured.AnalysisException: Not allowed to create a permanent view `spark_catalog`.`default`.`revenue0` by referencing a temporary view lineitem. Please create a temp view instead by CREATE TEMP VIEW.
```

I need to manually change to 
```suggestion
create temp view revenue0 (supplier_no, total_revenue) as
```

Reply:
Rather than change the original SQL, I pushed a commit https://github.com/apache/datafusion-benchmarks/pull/1/commits/ec38aacc7df1a44cca8cb4a7b0efd8f05113f701 to have the comet runner replace `create view` with `create temp view` when reading the queries